eng/Versions.props

```diff
@@ -6,7 +6,7 @@
     <MajorVersion>6</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
-    <PreReleaseVersionLabel>alpha</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>llvm</PreReleaseVersionLabel>
```

Comment:
I have realized that this won't be sufficient to avoid collisions given how nuget works and how things are setup currently.

The problem is what `PackageReference Include="Microsoft.DotNet.ILCompiler" Version="6.0.0-*"` package reference is going to pick up the latest `*` that will be `llvm` once this starts publishing.

I think it will be best to use different package name for this branch, like `Microsoft.DotNet.ILCompiler.LLVM`. 

Could you please rename `Microsoft.DotNet.ILCompiler` to `Microsoft.DotNet.ILCompiler.LLVM` under `src\installer\pkg\projects` ? (Both directory names and references in sources.)

There may be a few other places througout the tree that may need similar rename.

Reply:
Sure, is the label change now superfluous ?

Reply:
Yes - if you change the package name, it is not necessary to change the label.

Reply:
Maybe I've gone overboard on this, I renamed the targets file and the pkgproj.